spring-integration-core/src/main/java/org/springframework/integration/support/utils/IntegrationUtils.java

```diff
@@ -71,22 +71,21 @@ public static MessageBuilderFactory getMessageBuilderFactory(BeanFactory beanFac
 		MessageBuilderFactory messageBuilderFactory = null;
 		if (beanFactory != null) {
 			try {
-				 messageBuilderFactory = beanFactory.getBean(
+				messageBuilderFactory = beanFactory.getBean(
 						INTEGRATION_MESSAGE_BUILDER_FACTORY_BEAN_NAME, MessageBuilderFactory.class);
 			}
 			catch (Exception e) {
 				if (logger.isDebugEnabled()) {
-					logger.debug("No MessageBuilderFactory with name '"
-								+ INTEGRATION_MESSAGE_BUILDER_FACTORY_BEAN_NAME
-								+ "' found: " + e.getMessage()
-								+ ", using default.");
+					logger.debug("No MessageBuilderFactory with name '" +
+							INTEGRATION_MESSAGE_BUILDER_FACTORY_BEAN_NAME +
+							"' found: " + e.getMessage() +
+							", using default.");
```

Comment:
https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#line-wrapping:
>  Put the separator symbols at the end of the line rather on the next line 

But, eh, that was my own decision. 
I can come back to your preferences.
And we really can switch on http://checkstyle.sourceforge.net/config_whitespace.html#OperatorWrap checkstyle :smile: